.ghal.rules.json

```diff
@@ -42,6 +42,9 @@
           "(?i)dotnet-roslyn-api$": {
             "labels-add": ":books: Area - Roslyn API Reference,:file_folder: Repo - roslyn-api-docs"
           },
+          "(?i)dotnet-whats-new$": {
+            "labels-add": ":bookmark_tabs: Information Architecture"
```

Comment:
should we create a new area label for this since each folder has its own? Or too much for such a small area?

Information Architecture was the name that Wade gave to internal priorities activities.

Reply:
Good point. I was trying to avoid making more labels (especially after trimming them). I do think this is a good use case for anything related to What's new. It should also help us find the PR each month. New label is [what's new](https://github.com/dotnet/docs/labels/%3Astar2%3A%20What%27s%20New)